HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -0,0 +1,184 @@
+##Check the Use column on the right for the devices you want to combine.
+::
+Now you need to tell SuperCollider to use your new aggregate device.
+code::
+Server.default.options.device = "Aggregate Device"; //or the name you have specified in the previous step
+::
+
+After (re)booting the server (code::Server.default.reboot::) you should see in the post window that it now uses the Aggregate Device instead of system defaults:
+teletype::
+[...]
+"Aggregate Device" Input Device
+[...]
+"Aggregate Device" Output Device
+::
+
+Note that when you specify a sound device through SuperCollider's code::ServerOptions::, there is no need to use the aggregate device as the system's default device.
+
+
+section:: Linux
+
+By default SuperCollider on Linux uses JACK, and the audio device selection is managed by the JACK server. teletype::ServerOptions:: cannot override JACK's selection of audio hardware.
+
+subsection:: Setup with JACK server
+note::The SuperCollider server is considered a JACK emphasis::client::. In the following paragraph the term emphasis::client:: will refer to SuperCollider server, from the perspective of JACK. ::
+
+When the server is compiled to use JACK as audio backend, the code::ServerOption::'s teletype::device:: can be used in two ways:
+list::
+## to set the client name to register with JACK:
+code::
+Server.default.options.device = "my_synth";
+::
+## to use a specific JACK server, as well as set the client name:
+code::
+Server.default.options.device = "JACKServerName:scsynthName";
+::
+::
+A code::nil:: device is equivalent to code::Server.default.options.device = "default:SuperCollider";::
+
+The JACK connections can be configured via the environment variables teletype::SC_JACK_DEFAULT_INPUTS:: and
+teletype::SC_JACK_DEFAULT_OUTPUTS::. The format is either a string that specifies another jack client or a comma-separated list of jack ports.
+
+code::
+// connect first to input channels with system
+"SC_JACK_DEFAULT_INPUTS".setenv("system:capture_1,system:capture_2");
+
+// connect all output channels with system
+"SC_JACK_DEFAULT_OUTPUTS".setenv("system");
+::
+
+section:: Windows
+
+By default the server will boot to your system's default audio devices using teletype::MME:: driver (which usually means higher latency).
+
+On Windows there are multiple audio driver APIs (e.g. teletype::MME::, teletype::WASAPI::, teletype::ASIO:: etc.) that can be used to communicate with audio devices. The API (listed before the device name) needs to match between the input and the output, for example:
+code::
+o = Server.default.options;
+o.inDevice_("Windows WASAPI : Microphone");
+o.outDevice_("Windows WASAPI : Speakers");
+Server.default.reboot;
+::
+On Windows you strong::cannot:: programmatically obtain the list of available audio devices. However, all the devices are listed in the post window when the server is booting: code::Server.default.boot::
```

Comment:
just to be clear -- you _can_ get this list, just not in sclang, right?

Reply:
Correct - not in sclang nor in scsynth. From the sclang user perspective you can't get it with the software we're providing, that's why this wording. Is that reasonable?

Of course you can get it from the PortAudio backend, that's what scsynth does to list devices in the console.

Reply:
I would just add "available audio devices _in sclang_." at the end of that sentence. Otherwise it sounds a bit confusing, like it's fundamentally impossible (which wouldn't surprise me with Windows :) ).